sdk/identity/Azure.Identity/src/DefaultAzureCredential.cs

```diff
@@ -113,13 +113,18 @@ private async Task<AccessToken> GetTokenAsync(bool isAsync, TokenRequestContext
                 }
                 else
                 {
-                    exceptions.Add(exToken.Exception);
-
-                    throw scope.Failed(AuthenticationFailedException.CreateAggregateException($"{UnhandledExceptionMessage} {_sources[i].GetType().Name} failed with unhandled exception {exToken.Exception.Message}.", new ReadOnlyMemory<object>(_sources, 0, i + 1), exceptions));
+                    throw scope.Failed(new AuthenticationFailedException(UnhandledExceptionMessage, exToken.Exception));
                 }
             }
 
-            throw scope.Failed(AuthenticationFailedException.CreateAggregateException(DefaultExceptionMessage, new ReadOnlyMemory<object>(_sources, 0, i), exceptions));
+            StringBuilder errorMsg = new StringBuilder(DefaultExceptionMessage).Append(" (");
```

Comment:
I personally would prefer no parenthesis.

Something like this:

Exception: Reasons, Exception: Reasons, Exception: Reasons

Just to reduce the size of the message even further.

Reply:
I used the parenthesis because this is what AggregateException does to denote the inner exceptions. But, honestly I don't feel strongly about it. I'll remove them.

Reply:
See my message above.